doc/make_atlas_table.py

```diff
@@ -0,0 +1,356 @@
+    fetch_atlas_difumo,
+    fetch_atlas_harvard_oxford,
+    fetch_atlas_juelich,
+    fetch_atlas_msdl,
+    fetch_atlas_pauli_2017,
+    fetch_atlas_schaefer_2018,
+    fetch_atlas_smith_2009,
+    fetch_atlas_surf_destrieux,
+    fetch_atlas_talairach,
+    fetch_atlas_yeo_2011,
+    load_fsaverage,
+    load_fsaverage_data,
+)
+from nilearn.maskers import (
+    NiftiLabelsMasker,
+    NiftiMapsMasker,
+    SurfaceLabelsMasker,
+)
+from nilearn.plotting import plot_prob_atlas, plot_roi, plot_surf_roi
+from nilearn.surface import SurfaceImage
+
+
+def _update_dict(dict_for_df, name, fn, data, doc_dir, output_file, n_rois=1):
+    """Update dictionary to use to create dataframe of atlases."""
+    fn_name = fn.__name__.replace("fetch_atlas_", "")
+
+    dict_for_df["name"].append(
+        f"**{name.replace('_', ' ')}**<br>"
+        + f"*template*: {data.template}<br>"
+        + f"*number of regions*: {n_rois}<br>"
+        + "{ref}`description "
+        + f"<{fn_name}_atlas>`<br>"
+    )
+
+    dict_for_df["image"].append(
+        f"![name](../{output_file.relative_to(doc_dir)!s})"
+    )
+
+    return dict_for_df
+
+
+def _generate_markdown_file(filename, dict_for_df):
+    """Generate a markdown file with a table of atlases."""
+    atlas_table = pd.DataFrame(dict_for_df)
+    atlas_table = atlas_table.sort_values(by=["name"])
+
+    markdown_file = doc_dir / "modules" / filename
+    with markdown_file.open("w") as f:
+        f.write(f"""
+<!--
+!!!!! DO NOT EDIT MANUALLY !!!!!
+This file is auto-generated.
+To modify the content of this file do it via the script:
+{Path(__file__).name}
+-->
+""")
+
+        f.write("""
+```{warning}
+The atlases shipped with Nilearn
+do not necessarily use the same MNI template
+as the default MNI template used by Nilearn for plotting.
+
+This may lead to atlas being poorly coregistered
+to the underlay image used as underlay:
+the atlas can appear smaller or bigger than the brain.
+
+This can seen clearly on some of the images below.
+
+You also should not use maskers with an atlas
+that is not coregistered properly coregistered
+with the images you want to extract data from
+as this may lead to invalid results.
+```
+
+""")
+        atlas_table.to_markdown(buf=f, index=False)
+
+
+DEBUG = False
+GENERATE_FIG = True
+
+doc_dir = Path(__file__).parent
+output_dir = doc_dir / "images"
+
+plot_config = {
+    "draw_cross": False,
+    "colorbar": True,
+    "display_mode": "ortho",
+    "cut_coords": [0, 0, 0],
+}
+
+
+"""
+VOLUME DETERMINISTIC ATLASES
+"""
+
+# dict to define fetching options for each atlas
+deterministic_atlases = {
+    "AAL": {"fn": fetch_atlas_aal, "n_rois": [117, 167]},
+    "BASC multiscale (2015)": {
+        "fn": fetch_atlas_basc_multiscale_2015,
+        "params": {"resolution": 20, "version": "sym"},
+        "n_rois": [7, 12, 20, 36, 64, 122, 197, 325, 444],
+    },
+    "Destrieux (2009; volume)": {"fn": fetch_atlas_destrieux_2009},
+    "Harvard-Oxford": {
```

Comment:
```suggestion
    "Harvard-Oxford (cortical)": {
```

Just because we are specifying sub-cortical...